src/state/spacing.js

```diff
@@ -0,0 +1,22 @@
+import type { Position, Spacing } from '../types';
+
+export const add = (spacing1: Spacing, spacing2: Spacing): Spacing => ({
```

Comment:
at some point it will be good to add some tests for these functions, but for now it looks good

Reply:
i did :) https://github.com/atlassian/react-beautiful-dnd/blob/b53e27efd85ae746c351805c1e98d43438d0f1c1/test/unit/state/spacing.spec.js

Reply:
nice